packages/react-native-renderer/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "react-native-renderer",
+  "version": "15.0.0-rc.2",
+  "description": "React package for use inside react-native.",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/facebook/react.git"
+  },
+  "keywords": [
+    "react",
+    "react-native"
+  ],
+  "license": "BSD-3-Clause",
+  "bugs": {
+    "url": "https://github.com/facebook/react/issues"
+  },
+  "homepage": "https://facebook.github.io/react-native/",
+  "dependencies": {
+    "fbjs": "0.1.0-alpha.4",
```

Comment:
good call. This was actually long before fbjs was in RN. I should update this. @zpao Your warning didn't catch this. :)